plugins/kubernetes-backend/src/kubernetes-auth-translator/AwsIamKubernetesAuthTranslator.ts

```diff
@@ -32,8 +32,11 @@ const makeUrlSafe = pipe([replace('+', '-'), replace('/', '_')]);
 export class AwsIamKubernetesAuthTranslator
   implements KubernetesAuthTranslator {
   async getBearerToken(clusterName: string): Promise<string> {
-    const credentialProvider = defaultProvider();
-    const credentials = await credentialProvider();
+    const credentials = AWS.config.credentials;
+    if (!(credentials instanceof Credentials)) {
+      throw new Error('no credentials found');
```

Comment:
```suggestion
      throw new Error('no AWS credentials found.');
```

Just a tiny bit more helpful error message, perhaps.